force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +69,7 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        return (a == null) ? null : (a.equals(0) ? 'Zero' : (a > 0) ? 'Positive' : 'Negative');
```

Comment:
While ternary operators are a clever way to reduce code to a single line, they sometimes become less readable than traditional if statements. I would advise at least moving your null guard to its own if statement. 
```
if (a == null) {
   return null;
}
return (a == 0) ? 'Zero' : (a > 0) ? 'Positive' : 'Negative';
```